lib/iris/tests/__init__.py

```diff
@@ -91,6 +92,13 @@
     GRIB_AVAILABLE = True
 
 
+#: CDL history attribute regular expression pattern for ISO8601 timestamp.
+iso8601 = '(?P<lhs>\s*:history = ")' \
+    '(?P<timestamp>\d{4}\-?\d{2}\-?\d{2}' \
+    '(T\d{2}\:?\d{2}\:?\d{2}(\.\d{3})?(([\+\-]\d{2}\:?\d{2})|Z)?)?: )' \
+    '(?P<rhs>.*)'
```

Comment:
How do you get the string to pass to `strptime` in the first place? The timestamp is embedded within a sea of CDL, which is represented as a single string for the whole netCDF file ...